xDrip/Services/CloudSync/UploadRequestFactory.swift

```diff
@@ -122,4 +123,32 @@ final class UploadRequestFactory: UploadRequestFactoryLogic {
         
         return request
     }
+    
+    func createDeviceStatusRequest() -> URLRequest? {
+        let settings = User.current.settings.nightscoutSync
+        guard settings?.uploadBridgeBattery == true else { return nil }
+        guard let baseURLString = settings?.baseURL else { return nil }
+        guard let baseURL = URL(string: baseURLString) else { return nil }
+        guard let apiSecret = settings?.apiSecret else { return nil }
+        
+        let url = baseURL.appendingPathComponent("/api/v1/devicestatus")
+        var request = URLRequest(url: url)
+        
+        request.allHTTPHeaderFields = [
```

Comment:
This code is duplicated a lot in this factory, let's create some solution to unify that code.